portal-kernel/src/com/liferay/portal/kernel/service/persistence/BasePersistence.java

```diff
@@ -166,7 +164,7 @@ public T findByPrimaryKey(Serializable primaryKey)
 	 * @return the range of matching rows
 	 * @see    com.liferay.portal.kernel.dao.orm.QueryUtil#list(
 	 *         com.liferay.portal.kernel.dao.orm.Query,
-	 *         com.liferay.portal.kernel.dao.orm.Dialect, int, int)
+	 *         Dialect, int, int)
 	 */
 	public <V> List<V> findWithDynamicQuery(
 		DynamicQuery dynamicQuery, int start, int end);
```

Comment:
This is not a Commerce file. Can you please rebase so I can forward the PR? Thanks

Reply:
In progress